rtc-video-quality/generate_data.py

```diff
@@ -107,7 +107,6 @@ def clip_arg(clip):
 
 def psnr_to_dmos(score):
     # DMOS = 1 - 1 / (1 + math.exp((-0.1657) * (score + (-26.19))))
-    params = [-0.1657, -26.19]
     v = (1 - 1 / (1 + math.exp((-0.1657 * (score + -26.19)))))
     return v
```

Comment:
There's no need for "v" here. Also you've got a couple sets of unnecessary parens.
All you need is:
  return 1 - 1 / (1 + math.exp(-0.1657 * (score + -26.19)))